BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -28,7 +28,7 @@ int VerifyRsaSignature (CONST PUB_KEY_HDR *PubKeyHdr, CONST SIGNATURE_HDR *Signa
   Ipp8u  *rsa_n;
   Ipp8u  *rsa_e;
   Ipp16u  mod_len;
-  Ipp8u  bn_buf[3400];
+  Ipp8u  bn_buf[5200];

```

Comment:
Buf size for RSA3K

Reply:
This is too big to put on stack.  We can do it currently, but it might run into issue.  We don't have such a big stack in Stage1B.   If it is really big, we need to revisit.

Reply:
Noted.  Will check for dynamic allocation methods or other optimizations,